src/scss/shared/_validity.scss

```diff
@@ -5,6 +5,7 @@
 		*:valid:not(button):not(:disabled) {
 			color: _oFormsGet('valid-base');
 			border-color: _oFormsGet('valid-base');
+			border-width: 2px;
```

Comment:
At a glance looks like these `border-width`s would effect multiple form inputs, could you help me understand what they're needed for?

Reply:
I don't think they are needed, it was more for me to see the color better to see if it aligned with the design